### ISSUE TITLE
Config option for metric/imperial

### ISSUE THREAD

**Issue by Naesstrom** [bgulla/boozer#20]
Would it be possible to add a option to the config.ini to select either imperial or metric units. I'd rather have it in celcius and maybe liters for volume.

**Comment by bgulla** [comment]
Yeah definitely, that's not even that hard because the sensors actually read in as celsius and I convert them to imperial.
